lib/trino-parquet/src/main/java/io/trino/parquet/ParquetTypeUtils.java

```diff
@@ -264,4 +269,9 @@ public static Slice getLongDecimalValue(byte[] bytes)
         BigInteger value = new BigInteger(bytes);
         return unscaledDecimal(value);
     }
+
+    public static boolean isNegative(byte[] bytes, int offset)
+    {
+        return (bytes[offset] & 0x80) != 0;
```

Comment:
This seems like a convoluted way to say `bytes[offset] < 0`

Reply:
yes, i've simplified this in a separate commit now